src/live_restore/live_restore_private.h

```diff
@@ -0,0 +1,68 @@
+/*-
+ * Copyright (c) 2014-present MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *	All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
+
+#pragma once
+
+#define WT_LIVE_RESTORE_FS_TOMBSTONE_SUFFIX ".deleted"
+
+/*
+ * OFFSET_END returns the last byte used by a range (inclusive). i.e. if we have an offset=0 and
+ * length=1024 OFFSET_END returns 1023
+ */
+#define OFFSET_END(offset, len) (offset + (wt_off_t)len - 1)
```

Comment:
I think these macros need a WT_ in front of them even if they're in a _private.h file.

Reply:
d2c052ff9